lib/sqewer/local_connection.rb

```diff
@@ -9,6 +9,10 @@ def self.parse_queue_url(queue_url_starting_with_sqlite3_proto)
       raise "The scheme of the SQS queue URL should be with `sqlite3' but was %s" % uri.scheme
     end
 
+    if uri.hostname == 'memory'
+      return ['memory', nil]
```

Comment:
Suggestion to stay closer to the SQLite API:
```suggestion
      return [':memory:', nil]
```

Reply:
nice, much better! I could remove a hard-coded string because of this change